pkg/v1/tarball/write.go

```diff
@@ -293,6 +284,29 @@ func calculateManifest(refToImage map[name.Reference]v1.Image) (m Manifest, err
 	return m, nil
 }
 
+// CalculateSize calculates the expected size of the output tar file
+func CalculateSize(refToImage map[name.Reference]v1.Image) (size int64, err error) {
```

Comment:
Having both this and [`CalculateTarFileSize`](https://godoc.org/github.com/google/go-containerregistry/pkg/v1/tarball#CalculateTarFileSize) seems confusing. Can we rename one, or make it unexported?

Reply:
Sure, I can rename `CalculateTarFileSize()`. 5 mins.

Reply:
Done. Have a look.

Reply:
Ouch, this is still open. Can we get a review, merge it in, close it out?

Reply:
Oof, it's been a strange couple months so I've let some things slip, sorry about that.